src/main/java/androidx/content/Context.kt

```diff
@@ -91,3 +95,25 @@ inline fun Context.withStyledAttributes(
         typedArray.recycle()
     }
 }
+
+/**
+ * Return a [Bitmap] decoded from [Resources]
+ *
+ * @param resId Resource id of a drawable from `res/drawable` folder
+ * @param options Decoding options
+ */
+inline fun Resources.getBitmap(
+    @DrawableRes resId: Int,
+    options: BitmapFactory.Options? = null
+): Bitmap = BitmapFactory.decodeResource(this, resId, options)
+
+/**
+ * Return a [Bitmap] decoded from [Resources] of this [Context]
+ *
+ * @param resId Resource id of a drawable from `res/drawable` folder
+ * @param options Decoding options
+ */
+inline fun Context.getBitmap(
```

Comment:
Are all Resources extension functions in this library going to have matching Context extension functions? (@jake?)